specs-go/oci.go

```diff
@@ -37,15 +37,14 @@ func ApplyEditsToOCISpec(config *spec.Spec, edits *ContainerEdits) error {
 	}
 
 	if len(edits.Env) > 0 {
-
 		if config.Process == nil {
 			config.Process = &spec.Process{}
 		}
-
 		config.Process.Env = append(config.Process.Env, edits.Env...)
 	}
 
 	for _, d := range edits.DeviceNodes {
+		config.Linux.Devices = append(config.Linux.Devices, toOCILinuxDevice(d))
 		config.Mounts = append(config.Mounts, toOCIDevice(d))
```

Comment:
Should we remove the mounts? In my testing, it was not needed to mount the devices if they were specified as `LinuxDevices` along with their cgroups.

Reply:
+1

Reply:
done